style/style.css

```
@@ -102,3 +106,59 @@ body {
   color: var(--Dark-gray);
   font-weight: bold;
 }
+.support-information {
+  width: 750px;
+  padding: 20px 50px;
+  margin: 0 auto;
+  border-radius: 8px;
+  background-color: #ffff;
+  -webkit-box-shadow: 3px 4px 23px 10px rgba(209, 209, 209, 0.46);
+  -moz-box-shadow: 3px 4px 23px 10px rgba(209, 209, 209, 0.46);
+  box-shadow: 3px 4px 23px 10px rgba(209, 209, 209, 0.46);
+  display: flex;
+  flex-direction: column;
+}
+
+.cards-information {
+  display: flex;
+  justify-content: space-between;
+}
+
+.backed-information {
+  border-inline: 1px solid var(--Dark-gray-btn);
```

```suggestion
  border-inline: 1px solid var(--dark-gray-btn);
```

Normalmente a gente tem o padrãozinho de deixar tudo minusculo, só serve para ter um padrão mesmo, nada que influencie no resultado

---

Já resolvido